src/ImageSharp/Formats/Webp/AlphaDecoder.cs

```diff
@@ -307,13 +311,50 @@ private void ExtractAlphaRows(Vp8LDecoder dec)
 
         private static void HorizontalUnfilter(Span<byte> prev, Span<byte> input, Span<byte> dst, int width)
         {
-            byte pred = (byte)(prev == null ? 0 : prev[0]);
+#if SUPPORTS_RUNTIME_INTRINSICS
+            if (Sse2.IsSupported)
+            {
+                dst[0] = (byte)(input[0] + (prev == null ? 0 : prev[0]));
+                if (width <= 1)
+                {
+                    return;
+                }
+
+                int i;
+                var last = Vector128.Create(dst[0], 0, 0, 0);
```

Comment:
```suggestion
                var last = Vector128<int>.Zero.WithElement(0, dst[0]);
```
Results in less `vpinsrd` in the [machine code](https://sharplab.io/#v2:EYLgxg9gTgpgtADwGwBYA0AXEBDAzgWwB8ABAJgEYBYAKGIAYACY8gOgCUBXAOwwEt8YLAJI8ovLrl5hcAbho1iAZiakGAYQYBvGg11NlANRhgM0cqQAcAHnEYAfAwCy5ABQBlAA7YuV4AE8MGAcAE1wMAEoGHT1taj14pgB2BiMTM0sWNVhsQJdQjABtOgBdNAY6Mory8Lk4vQBfaN0m/RTjUyhza1sHR1J3Lx9/QJCwyJbYhL1iZNSOrpseOxYALRgoCBYAdV4MAAsAUQAbGAEeFyr8ouKalsbqeqA).

Reply:
Wow! This one surprises me!